src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+
+const DIFFICULTY = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameStart,setIsGameStart]=useState(false)
+  const startQuiz = async () => {
+    setIsLoading(true);
+    setIsGameStart(true)
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      DIFFICULTY.EASY
+    );
+    setQuestions(newQuestions);
+    setScore(0);
+    setUserAnswers([]);
+    setCurrentQuestionNumber(0);
+    setIsLoading(false);
+  };
+  const checkAnswer = (e) => {
+    if (isGameStart) {
+      const answer = e.currentTarget.value;
+      const answerAgainstCorrect = questions[currentQuestionNumber].correct_answer
```

Comment:
i'm not sure i understand what this variable name means. answerAgainstCorrect. think it needs to be renamed...

line 34 = the answer the user gave
line 35 = the objectively correct answer

rename both variables based on what they represent